docs-rtd/howto/manage-offers.md

```diff
@@ -78,14 +90,44 @@ resource "juju_integration" "this" {
 > See more: [`juju_integration` > `via`](https://registry.terraform.io/providers/juju/juju/latest/docs/resources/integration#via)
 
 
-## Remove an offer
-> Who: User with [offer `admin` access](https://juju.is/docs/juju/user-permissions#heading--offer-admin).
+(manage-access-to-an-offer)=
+## Manage access to an offer
 
-To remove an offer, in your Terraform plan, remove its resource definition.
+When using Juju alone, you can grant offer access only to a user. When using Juju with JAAS, you can grant offer access to a user, a group, or a service account.
 
-> See more: [`juju_offer`](https://registry.terraform.io/providers/juju/juju/latest/docs/resources/offer)
+### Using Juju alone
+When using Juju alone, to grant a user access to an offer, in your Terraform plan add a `juju_access_offer` resource, specifying the offer URL and setting the Juju access level to the list of users you want to grant that level. For example:
```

Comment:
a suggestion for all Juju VS JAAS sections:
"When applying a Terraform plan to a Juju controller"
vs
"When applying a Terraform plan to a JAAS controller"

Because when you say "When using Juju alone" it sounds to me like you're using the Juju CLI, not the provider.